File: java/program/Elektronik.java
// Class Elektronik
public class Elektronik {
    private String id;
    private String nama;
    private int harga;

    // Default constructor  
    public Elektronik() {
        this.id = "";
        this.nama = "";
        this.harga = 0;
    }
    
    // Constructor with parameters
    public Elektronik(String id, String nama, int harga)
    {
        this.id = id;
        this.nama = nama;
        this.harga = harga;
    }

    // Getter & Setter for id
    public String getId() {return id;}
    public void setId(String id) {this.id = id;}

    // Getter & Setter for nama
    public String getNama() {return nama;}
    public void setNama(String nama) {this.nama = nama;}

    // Getter & Setter for harga
    public int getHarga() {return harga;}
    public void setHarga(int harga) {this.harga = harga;}
}

File: php/program/Elektronik.php
<?php
// Class Elektronik
class Elektronik {
    private $id;
    private $nama;
    private $harga;
    private $image; 

    // Constructor
    public function __construct($id = "", $nama = "", $harga = 0, $image = "") {
        $this->id = $id;
        $this->nama = $nama;
        $this->harga = $harga;
        $this->image = $image;
    }

    // Getter & Setter for id, nama, harga, image
    public function setId($id) { $this->id = $id; }
    public function getId() { return $this->id; }

    public function setNama($nama) { $this->nama = $nama; }
    public function getNama() { return $this->nama; }

    public function setHarga($harga) { $this->harga = $harga; }
    public function getHarga() { return $this->harga; }

    public function setImage($image) { $this->image = $image; }
    public function getImage() { return $this->image; }
}
